File: trendify.Server/trendify.Core/Models/Orders/DeliveryAddressModel.cs
namespace trendify.Core.Models.Orders
{
    public class DeliveryAddressModel
    {
        public string Name { get; set; } = null!;
        public string StreetAddress { get; set; } = null!;
        public string City { get; set; } = null!;
        public string ZipCode { get; set; } = null!;
    }
}

File: trendify.Client/src/app/models/order/deliveryAddress-model.ts
export class DeliveryAddressModel {
    name!: string;
    streetAddress!: string;
    city!: string;
    zipCode!: string;

    constructor(init?: Partial<DeliveryAddressModel>) {
        if (init) {
            Object.assign(this, init);
        }
    }
}
